plugins/gradle/example-client-kotlin/build.gradle

```diff
@@ -0,0 +1,63 @@
+import com.kobylynskyi.graphql.codegen.model.GeneratedLanguage
+import io.github.kobylynskyi.graphql.codegen.gradle.GraphQLCodegenGradleTask
+
+plugins {
+    id 'java'
+    id "org.jetbrains.kotlin.jvm" version "1.3.71"
+    id "io.github.kobylynskyi.graphql.codegen" version "4.0.2-SNAPSHOT"
+}
+
+group 'io.github.dreamylost'
+version '1.0-SNAPSHOT'
```

Comment:
Let's keep the version of all example projects in sync with graphql-codegen version (4.0.2-SNAPSHOT)